examples/cf-for-k8s-v0.2.0-custom/config.yml

```diff
@@ -10777,7 +10777,7 @@ data:
             require_client_certificate: true
 
 
-      # Manual 'whitebox' mode
+      # Manual 'open-box' mode
```

Comment:
Maybe we should include the complete examples directory in the wokeignorefile? (As almost all the the config files are generated through some instructions and we don't have much control over it)

I am fine with this approach as well :)

Reply:
@praveenrewar Good question. I don't have a strong opinion on this. Since you're a primary maintainer of this repo, I'll leave it to you to decide on the long-term plan. I'm going to merge this PR as is. Thanks for the review!